src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/feedback/feedback.component.html

```diff
@@ -0,0 +1,37 @@
+<div>
+  <form [formGroup]="feedbackForm" (ngSubmit)="onSubmit()">
+    <div>
+      <label for="project">Project: </label>
+      <select formControlName="project" class="form-control">
+        <option disabled>Select Project</option>
+        <option>Choose Project</option>
+        <option *ngFor="let projectName of project">{{projectName}}</option>
+      </select>
+      <div *ngIf="f.project.touched && f.project.invalid" class="alert alert-danger">
+        <div *ngIf="f.project.errors.required">Project is required.</div>
+      </div>
+    </div>
+    <div>
+      <label for="tracker">Tracker: </label>
+      <select formControlName="tracker" class="form-control">
+        <option disabled>Select Tracker</option>
+        <option>Choose Tracker</option>
+        <option *ngFor="let trackerName of tracker">{{trackerName}}</option>
+      </select>
+      <div *ngIf="f.tracker.touched && f.tracker.invalid" class="alert alert-danger">
+        <div *ngIf="f.tracker.errors.required">Tracker is required.</div>
+      </div>
+    </div>
+    <div>
+      <label for="subject">Subject: </label>
+      <input id="subject" type="text" formControlName="subject" class="form-control" placeholder="Add issue Title">
+    </div>
+    <div>
+      <label for="description">Description: </label>
+      <input id="description" type="text" formControlName="description" class="form-control" placeholder="Add issue description">
+    </div>
+    <div>
+      <button type="submit" class="btn btn-primary">Submit</button>
+    </div>
+  </form>
```

Comment:
You can use the modal component and call this form inside it. This modal service call can be used for it https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/frontend/src/app/shared/services/modal.service.ts.

You can get inspiration from this file: https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-flags-modal/osd-flags-modal.component.html